### PR TITLE
[MINOR] Add evoked GFP to tutorials

### LINKED ISSUE #4002: ENH: Allow retrieval of GFP
#2538 added the ability to plot GFPs. Currently, the GFP is [only temporarily calculated for plotting](https://github.com/Eric89GXL/mne-python/blob/7f8c69bae49041bb4f0507539ccda1bda7f0b394/mne/viz/evoked.py#L397), and the user has no easy way to access the data. 

 In our EEG workflow, we typically calculate GFPs for every single participant and condition, and average conditions across participants for plotting; or we compute statistics based on the GFP differences. It is therefore highly important for us to have easy access to the GFPs. We resorted to doing the calculations manually based on `Evoked.data`, but this is cumbersome as one has to "leave" the MNE sphere and implement the operations by hand via NumPy and/or Pandas -- which is not easy for beginners and error-prone, as Pandas by default [uses the unbiased estimator](http://stackoverflow.com/questions/24984178/different-std-in-pandas-vs-numpy) for standard deviation and NumPy doesn't.

I can try to implement a GFP function, but I would need assistance in doing so. I don't really know where to start or where to put that code: should it be a method of the `Evoked` class? Potentially exposed as a property, so it could be accessed via `Evoked.gfp`? Or should it be an entirely new class? Would it have to have its own plotting method? etc. pp. Any help and suggestions would be greatly appreciated.

### PR DESCRIPTION
**Pull request by jona-sassenhagen**
This adds a few lines to the evoked object tutorial to show how to extract and plot the SD across channels, as discussed in #4002 

I'm trying to close old issues :)

Closes #4002

### CODE REVIEW

File: tutorials/plot_object_evoked.py [hunk 1]
```diff
@@ -63,6 +63,12 @@
 print('Data from channel {0}:'.format(evoked.ch_names[10]))
 print(data[10])
 
+###############################################################################
+# In the same vein, we can quickly extract (and, e.g., plot) the GFP as the
+# standard deviation across channels.
+gfp = evoked.copy().pick_types(eeg=True, meg=False).data.std(0)
+plt.plot(evoked.times, gfp)
```
Comment (agramfort) [inline tutorials/plot_object_evoked.py]: can you add plt.xlabel and plt.ylabel? thanks

**Comment by larsoner** [comment]
https://10550-1301584-gh.circle-artifacts.com/0/html/auto_tutorials/plot_object_evoked.html

File: tutorials/plot_object_evoked.py [hunk 2]
```diff
@@ -63,6 +65,16 @@
 print('Data from channel {0}:'.format(evoked.ch_names[10]))
 print(data[10])
 
+###############################################################################
+# In the same vein, we can quickly extract (and, e.g., plot) the GFP as the
+# standard deviation across channels, here shown just for EEG.
+
+gfp = evoked.copy().pick_types(eeg=True, meg=False).data.std(axis=0)
+fig, ax = plt.subplots(1)
```
Comment (mmagnuski) [inline tutorials/plot_object_evoked.py]: you don't need the `1`, right?

Comment (larsoner) [reply tutorials/plot_object_evoked.py]: yes it's optional

**Comment by agramfort** [comment]
+1 for MRG

**Comment by jona-sassenhagen** [comment]
Ah I forgot the scaling. Good like this?

**Comment by massich** [comment]
a bit off topic question: Shall we add the definition of global field power (GFP) to the glossary? and link to it?

Does anyone see value to that? or all users know the terms enought that is not worth the effort?

**Comment by jona-sassenhagen** [comment]
No, that makes sense to me. You can do it directly on this PR?


________________________________

File: tutorials/plot_object_evoked.py [hunk 3]
```diff
@@ -8,10 +8,12 @@
 averaged data over trials. In MNE the evoked objects are usually created by
 averaging epochs data with :func:`mne.Epochs.average`.
 """
+# sphinx_gallery_thumbnail_number = 2
 
 import os.path as op
 
 import mne
+import matplotlib.pyplot as plt
```
Comment (agramfort) [inline tutorials/plot_object_evoked.py]: import mne last.

**Comment by agramfort** [comment]
thx @jona-sassenhagen

**Pull request merged**